README.md

```diff
@@ -119,6 +117,11 @@ Container images are built using Docker Compose, with the same images used to ru
 
 Use Docker Compose to run service locally.
 
+The service uses [Liquibase](https://www.liquibase.org/) to manage database migrations. To ensure the appropriate migrations have been run the utility script `scripts/start` may be run to execute the migrations, then the application.
+
+Alternatively the steps can be run manually:
+* run migrations
+  * `docker-compose -f docker-compose.migrate.yaml run database-up`
```

Comment:
If you add the `--rm` flag then the liquibase container is removed when the migration is complete so you don't get a build up of stopped liquibase containers.

```suggestion
  * `docker-compose -f docker-compose.migrate.yaml run --rm database-up`
```